terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -70,27 +70,41 @@ export function resetRunnersCaches() {
   ssmParametersCache.reset();
 }
 
+// Helper function to get the current AWS account ID
+async function getCurrentAccountId(): Promise<string> {
+  const sts = new STS();
+  const identity = await sts.getCallerIdentity().promise();
+  if (!identity.Account) {
+    throw new Error('Unable to retrieve AWS account ID');
```

Comment:
Instead of failing with error, should we just log the error and fallback to Amazon only images ?